src/Swoole/Request.php

```diff
@@ -280,6 +272,31 @@ public function getHeader(string $key, string $default = ''): string
         return $this->swoole->header[$key] ?? $default;
     }
 
+    /**
+     * Method for adding HTTP header parameters.
+     *
+     * @param  string  $key
+     * @param  string  $value
+     * @return void
+     */
+    public function addHeader(string $key, string $value): void
```

Comment:
Forgot to mention, we usually return $this for adding method changing where possible. Should also be fixed in utopia/fw and other methods if needed.